package.json

```diff
@@ -929,30 +929,29 @@
   },
   "scripts": {
     "prepare": "npm run download:artifacts && cd src/test/e2e && npm install",
-    "compile": "npm run check-types && npm run lint && node esbuild.js",
-    "compile:prod": "npm run check-types && npm run lint && node esbuild.js --production",
+    "compile": "npm run check-types && npm run lint && node esbuild.mjs",
+    "compile:prod": "npm run check-types && npm run lint && node esbuild.mjs --production",
     "compile:tests": "tsc -p .",
     "watch": "npm-run-all -p watch:esbuild watch:tsc",
-    "watch:esbuild": "node esbuild.js --watch",
+    "watch:esbuild": "node esbuild.mjs --watch",
     "watch:tsc": "tsc --noEmit --watch",
     "watch:tests": "tsc -p . -w",
     "watch:web": "npm-run-all -p watch:web:*",
-    "watch:web:esbuild": "node esbuild.js --watch",
+    "watch:web:esbuild": "node esbuild.mjs --watch",
     "watch:web:tsc": "tsc --noEmit --watch",
     "web": "npm run compile && vscode-test-web --browserType=chromium --extensionDevelopmentPath=. ./src/test/fixtures",
     "web:serve": "npx serve --cors -l 5001",
     "web:tunnel": "npx localtunnel -p 5001",
-    "download:artifacts": "ts-node ./build/downloader.ts",
+    "download:artifacts": "node ./build/downloader.mjs",
     "vscode:prepublish": "npm run compile:prod",
     "package": "vsce package",
     "pretest": "npm run compile:tests && npm run compile && npm run lint",
     "test": "vscode-test",
     "wdio": "npm run compile && wdio run ./src/test/e2e/wdio.conf.ts",
-    "lint": "eslint src --ext ts",
-    "prettier": "prettier \"**/*.+(js|json|ts)\"",
```

Comment:
I added markdown to the ignore to prevent touching the docs, but thought since the changes to the yaml were along best practices (tabs, spaces, quoted strings, etc) it was best to include them